scripts/easy-spinnaker/check_inst_complete.sh

```diff
@@ -39,7 +39,7 @@ echo "==========================================================================
 echo 
 echo "Installation of Spinnaker is now complete. Login to the URL below using admin/OpsMx@123"
 echo 
-kubectl get svc spin-deck-ui -n oes -o jsonpath='{"http://10.168.3.10:"}{..nodePort}{"\n"}'
+kubectl get svc spin-deck-np -n oes -o jsonpath='{"http://10.168.3.10:"}{..nodePort}{"\n"}'
```

Comment:
The latest OES creates spin-deck-ui. Why is this being changed back to spin-deck-np?

Reply:
Fixed